pkg/githubutils/githubutils.go

```diff
@@ -0,0 +1,74 @@
+package githubutils
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/grafana/plugin-validator/pkg/logme"
+)
+
+func FetchLatestGrafanaSdkRelease() (Release, error) {
+	return FetchGrafanaSdkReleaseByTag("latest")
+}
+
+func FetchGrafanaSdkReleaseByTag(tag string) (Release, error) {
```

Comment:
I now this is fairly simple at the moment but have you considered using a library for this? like https://github.com/google/go-github. It should help handling weird cases.

Reply:
Also, I don't know if we are able to use any persistence with the validator but it may be worth to cache these results so we don't need to query them again (not a big problem if we use a token though).

Reply:
I was planning to use the go-github but, yes, this is such a simple http request that using a whole library makes for more work and yet another dependency.

If we start adding more checks that rely on the github API we should move to it though.